### PR TITLE
CLI: Add `verdi devel launch-multiply-add`

### PR DESCRIPTION
**Pull request by GeigerJ2**
For us `devs` (@agoscinski, @khsrali) to also debug WorkChains, and/or get those into a fresh profile, akin to `verdi devel launch-add`.

### CODE REVIEW

File: tests/cmdline/commands/test_devel.py [hunk 1]
```diff
@@ -56,3 +56,38 @@ def test_launch_add_code(run_cli_command, aiida_code_installed):
...
+
+
+@pytest.mark.usefixtures('started_daemon_client')
+def test_launch_multiply_add_daemon(run_cli_command, submit_and_await):
+    """Test ``verdi devel launch-multiply-add`` with the ``--daemon`` flag."""
+    result = run_cli_command(cmd_devel.devel_launch_multiply_add, ['--daemon'])
+    assert re.search(r'Submitted workflow', result.stdout)
+
+    node = QueryBuilder().append(ProcessNode, tag='node').order_by({'node': {'ctime': 'desc'}}).first(flat=True)
+    submit_and_await(node)
```
Comment (agoscinski) [inline tests/cmdline/commands/test_devel.py:82]: very useful command, did not know about it

File: src/aiida/cmdline/commands/cmd_devel.py [hunk 2]
```diff
@@ -180,6 +180,57 @@ def devel_launch_arithmetic_add(code, daemon, sleep):
...
+@options.CODE(type=types.CodeParamType(entry_point='core.arithmetic.add'))
+@click.option('-d', '--daemon', is_flag=True, help='Submit to the daemon instead of running blockingly.')
+def devel_launch_multiply_add(code, daemon):
+    """Launch a ``MultipylAddWorkChain``.
+
+    Unless specified with the option ``--code``, a suitable ``Code`` is automatically setup. By default the command
+    configures ``bash`` on the ``localhost``. If the localhost is not yet configured as a ``Computer``, that is also
+    done automatically.
+    """
+    from shutil import which
```
Comment (agoscinski) [inline src/aiida/cmdline/commands/cmd_devel.py:193]: did not know about it, useful

**Review by agoscinski — COMMENTED** [review]
Looks good, not sure why test fail

File: tests/cmdline/commands/test_devel.py (hunk 1 above)
Comment (GeigerJ2) [reply tests/cmdline/commands/test_devel.py:82]: Literally copy-pasted everything from the tests for `verdi devel launch-add` :D

**Comment by GeigerJ2** [comment]
> Looks good, not sure why test fail

Story of my life :D

**Comment by GeigerJ2** [comment]
Pls approve when CI is done, @agoscinski, pretty please :3

**Review by agoscinski — APPROVED** [review]

**Pull request merged**